modules/virt-enabling-disabling-vm-delete-protection-web.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the web console
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the {product-title} web console. Should you change your mind, you can easily disable delete protection for a VM.
+
+By default, VMs are not enabled with delete protection. You must set the option for each individual VM.
+
+.Prerequisites
+
+* You have installed {VirtProductName} 4.19 or later.
+
+.Procedure
+
+. From the {product-title} web console, choose your view:
+
+    * For a virtualization-focused view, select *Administrator* → *Virtualization* → *VirtualMachines*.
+
+    * For a general view, navigate to *Virtualization* → *VirtualMachines*.
+
+. From the *VirtualMachines* list, select the VM whose delete protection you want to enable or disable.
+
+. Click the *Configuration* tab.
+
+. In the *VirtualMachines details*, choose to enable or disable the protection as follows:
+
+* To enable the protection:
+.. Set the *Deletion protection* switch to *On*.
+.. Click *Enable* to confirm the protection.
+
+ * To disable the protection:
+.. Set the *Deletion protection* switch to *Off*.
+.. Click *Disable* to confirm the protection.
```

Comment:
Instead of "confirm the protection" should this be something like "disable the protection" or "remove the protection"?